src/builder.rs

```diff
@@ -93,11 +84,15 @@ impl Builder {
         self
     }
 
+    /// Build with default rng.
     pub fn build<'a>(&self) -> Sim<'a> {
         self.build_with_rng(Box::new(rand::rngs::SmallRng::from_entropy()))
     }
 
-    pub fn build_with_rng<'a>(&self, rng: Box<dyn RngCore>) -> Sim<'a> {
+    /// Build a sim with a provided `rng`.
+    ///
+    /// This allows setting the random number generator used to fuzz
+    fn build_with_rng<'a>(&self, rng: Box<dyn RngCore>) -> Sim<'a> {
```

Comment:
Hey, just so you know, the function was public before and isn't anymore, is that by design?

Reply:
No, this was a mistake. It should be fixed after #152 is merged